flepimop/gempyor_pkg/src/gempyor/outcomes.py

```diff
@@ -444,22 +439,15 @@ def compute_all_multioutcomes(
                     )  # one draw per subpop
                 durations = np.repeat(durations[:, np.newaxis], len(dates), axis=1).T  # duplicate in time
                 durations = np.round(durations).astype(int)
-
-                hpar = pd.concat(
-                    [
-                        hpar,
-                        pd.DataFrame.from_dict(
-                            {
-                                "subpop": modinf.subpop_struct.subpop_names,
-                                "quantity": ["duration"] * len(modinf.subpop_struct.subpop_names),
-                                "outcome": [new_comp] * len(modinf.subpop_struct.subpop_names),
-                                "value": durations[0] * np.ones(len(modinf.subpop_struct.subpop_names)),
-                            }
-                        ),
-                    ],
-                    axis=0,
+                hpar = pd.DataFrame(
+                    data={
```

Comment:
'data=' may not be needed but it's okay, thanks!

Reply:
Yeah, it does not need to be provided as a keyword arg, but `data` does have a default value so I did so anyways for clarity ([link to docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html)). I'm fine with changing it if you feel strongly about it?

Reply:
> Yeah, it does not need to be provided as a keyword arg, but `data` does have a default value so I did so anyways for clarity ([link to docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html)). I'm fine with changing it if you feel strongly about it?

No need to change it, thanks!